Makefile

```diff
@@ -86,12 +86,48 @@ image: test deps
 	docker build -t \
 		$(KEPLER_IMAGE) \
 		--platform=linux/$(GOARCH) .
+	$(call docker_tag,$(KEPLER_IMAGE),$(ADDITIONAL_TAGS))
 
 # Push Docker image
 push:
-	docker push $(KEPLER_IMAGE)
+	$(call docker_push,$(KEPLER_IMAGE),$(ADDITIONAL_TAGS))
 
 # Run the application
 run:
 	$(BINARY_DIR)/$(BINARY_NAME)
 
+# docker_tag accepts an image:tag and a list of additional tags comma-separated
+# it tags the image with the additional tags
+# E.g. given foo:bar, a,b,c, it will tag foo:bar as foo:a, foo:b, foo:c
+define docker_tag
+@{ \
+	set -eu ;\
+	img="$(1)" ;\
+	tags="$(2)" ;\
+	echo "tagging container image $$img with additional tags: '$$tags'" ;\
+	\
+	img_path=$${img%:*} ;\
+	for tag in $$(echo $$tags | tr -s , ' ' ); do \
```

Comment:
In bash, it should handle an empty string for tags gracefully, I think it's good idea to include a check to handle whitespaced (and empty) strings because that might cause the loop to still execute and push an image with a tag that is a space.

Reply:
Good suggestion. but ROI is low IMHO. 
How about as a follow up PR so that we this functionality can go in?